languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+
+def whitelist_reader(path):
+    """Reads in whitelist file"""
+    with open(path, "r") as source:
+        for line in source:
+            yield line.rstrip().split()[0]
+
+
+def filter(tsv_path, phonemes, output_path):
+    """Creates tsv filtered by whitelist"""
+    with open(tsv_path, "r") as source:
```

Comment:
You can put two `with` statements on a single line, if under 80 lines. This will look something like:

`with open(tsv_path, "r") as source, open(output_path, "w") as output:`

Then adjust the indentation below.